robotframework_reportportal/model.py

```diff
@@ -227,3 +229,29 @@ def __init__(self, message):
     def __repr__(self):
         """Return string representation of the object."""
         return self.message
+
+    def handle_img_attachment(self):
```

Comment:
It should not be here. Moreover, the implementation can be done much simpler. Look into the pathlib library.